libs/service-portal/core/src/screens/NoDataScreen/NoDataScreen.tsx

```diff
@@ -7,98 +7,139 @@ import {
+      </>
+    )
+  }
   return (
-    <GridRow>
-      <GridColumn span={['1/1', '10/12']} offset={['0', '0']}>
-        <Box
-          marginTop={6}
-          marginBottom={6}
-          textAlign="center"
-          justifyContent="center"
-        >
-          {tag && (
-            <Box marginBottom={4}>
-              <Tag variant={tagVariant}>{tag}</Tag>
-            </Box>
-          )}
-          <Text variant="h1" as="h1" marginBottom={3}>
-            {title}
-          </Text>
-          <Text variant="default" as="div">
-            {children}
-          </Text>
-          {button && (
+    <>
+      <GridRow>
+        <GridColumn span={['1/1', '6/12']} offset={['0', '3/12']}>
+          <Box
+            marginTop={[3, 6]}
+            marginBottom={6}
+            textAlign="center"
+            justifyContent="center"
+          >
+            {tag && (
+              <Box marginBottom={4}>
+                <Tag variant={tagVariant}>{tag}</Tag>
+              </Box>
+            )}
+            <Text variant="h1" as="h1" marginBottom={3}>
+              {title}
+            </Text>
+            <Text variant="default" as="div">
+              {children}
+            </Text>
             <Box
-              marginTop={3}
-              marginBottom={3}
               display="flex"
-              textAlign="center"
+              flexDirection={['column', 'row']}
               justifyContent="center"
+              alignItems="center"
+              marginTop={4}
             >
-              {button.internal ? (
-                <Link to={button.link}>
-                  <Button
-                    variant={button.variant}
-                    size="small"
-                    icon="receipt"
-                    iconType="outline"
-                  >
-                    {button.text}
-                  </Button>
-                </Link>
-              ) : (
-                <a href={button.link}>
-                  <Button
-                    variant={button.variant}
-                    size="small"
-                    icon="receipt"
-                    iconType="outline"
-                  >
-                    {button?.text}
-                  </Button>
-                </a>
+              {button && (
+                <Box display="flex" justifyContent="center" alignItems="center">
+                  {renderButton(button)}
+                </Box>
+              )}
+              {secondaryButton && (
+                <Box
+                  marginLeft={[0, 2]}
+                  marginTop={[3, 0]}
+                  display="flex"
+                  textAlign="center"
+                  justifyContent="center"
+                >
+                  {renderButton(secondaryButton)}
+                </Box>
               )}
             </Box>
-          )}
-        </Box>
-      </GridColumn>
-
-      <GridColumn span={['1/1', '4/12']} offset={['0', '3/12']}>
-        <Box display="flex" justifyContent="center">
```

Comment:
Can get rid of the fragment, GridRows and GridColumns since it's just one column now?

Reply:
yes 💯  ✅